packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html

```diff
@@ -7,22 +7,22 @@
 <body>
     <font face="Helvetica">
         <h3>
-            Welcome to .NET Core!
+            Installation Completed Successfully Welcome to .NET Core!
```

Comment:
Neither "completed" nor "successfully" should be capitalized. There also needs to be some punctuation between the two parts to this.

The following edit would address my feedback, but please get the text reviewed doc writers.
```
Installation completed successfully. Welcome to .NET Core!
```